scrapscript.py

```diff
@@ -3400,19 +3442,32 @@ def test_match_record_spread(self) -> None:
             Int(4),
         )
 
-    def test_match_expr_as_boolean_symbols(self) -> None:
+    def test_match_expr_as_boolean_variants(self) -> None:
         self.assertEqual(
             self._run(
                 """
         say (1 < 2)
         . say =
-          | #false -> "oh no"
-          | #true -> "omg"
+          | #false () -> "oh no"
```

Comment:
Do you want to have syntactic sugar for #true and #false so that the hole is not required? I don't have a strong opinion. Would be nice to require the hole for consistency, but would also be nice to maintain back-compat with scrapscript programs written before..

Reply:
I don't know how parse an optional value here... if you have suggestions I'm all ears but otherwise I am fine to require updating the 5-10 scrapscript programs in existence